spec/support/jasmine-browser.json

```diff
@@ -0,0 +1,25 @@
+{
+  "srcDir": "spec/dummy/public/assets",
+  "srcFiles": [
+    "govuk_publishing_components/dependencies-*.js",
+    "govuk_publishing_components/all_components-*.js",
+    "govuk_publishing_components/modules-*.js",
+    "govuk_publishing_components/analytics-*.js",
+    "component_guide/accessibility-test-*.js",
+    "component_guide/filter-components-*.js"
+  ],
+  "specDir": "spec/javascripts",
+  "specFiles": [
+    "**/*[sS]pec.js"
+  ],
+  "helpers": [
+    "helpers/*.js",
+    "vendor/*.js"
+  ],
+  "env": {
+    "stopSpecOnExpectationFailure": false,
+    "stopOnSpecFailure": false,
+    "random": false
+  },
```

Comment:
I noticed that the defaults from the `jasmine-browser-runner` [Getting started guide](https://github.com/jasmine/jasmine-browser-runner#getting-started) are:

```json
  "env": {
    "random": true
  }
```

Is there a reason why we've diverged from the defaults? i.e. is it a deliberate choice, or has it happened by accident?

Reply:
I hope we don't need that. I turned it off with a rather cocky commit message after a few attempts locally worked fine. Hopefully build passes.

Reply:
And presumably `"stopSpecOnExpectationFailure": false` and `"stopOnSpecFailure": false` just allow the entire test suite to run despite errors, rather than failing fast at the first failure.

I guess that seems like a useful feature. Maybe that's why we've specifically enabled it?

Reply:
That's a leftover from me fixing a test that failed intermittently, but the fix got out in a separate PR and I forgot to switch back to true 🙈

Reply:
Ah yes these would be good to think about more, thanks for raising @ollietreend.

We'd want `"stopOnSpecFailure": false` in CI as we naturally want to know all the tests that fail. But if I understand `"stopSpecOnExpectationFailure": false` right - that'd be a test carrying on once an expectation fails which seems unusual to me. 

Looking at the repo though - the defaults are: https://github.com/jasmine/jasmine-browser-runner/blob/763ffc8a9b11e9aa621eb9bfe9709a1527f8bc89/lib/examples/default_config.json#L13-L17. These actually match what we have so we can probably just drop this block all together with an expectation that the upstream project remains sensible.

Reply:
The use of `"examples"` in that file path – `lib/examples/default_config.json` – leaves me suspicious that it's not the real source of truth for config defaults. It seems to be the template used when generating new config via `npm jasmine-browser-runner init` according to https://github.com/jasmine/jasmine-browser-runner/commit/22e32a6a2444ba6d82fd549955714e1409d90402.

Instead, I think Jasmine's defaults are actually defined in the main `jasmine-core` package here: https://github.com/jasmine/jasmine/blob/9e10743a1c7c0a0fc5696261af8b791d9dfe40ce/src/core/Env.js#L36-L134

Regardless, it looks like we were using the default config values, so they should be safe to remove! ✨

Reply:
Oh cool thanks for looking into that.